os/kernel/binary_manager/binary_manager.c

```diff
@@ -83,7 +83,7 @@ int binary_manager(int argc, char *argv[])
 	/* Scan user binary files and Register them */
 	binary_manager_scan_ubin();
 
-	ASSERT(binary_manager_get_kcount() > 0 && binary_manager_get_ucount() > 0);
+	ASSERT(binary_manager_get_ucount() > 0);
```

Comment:
@jeongarmy Could you give an opinion for this? Is binary manger working without kernel partition?

Reply:
Kernel partition should be registered for OTN.
So you need to change flash base address and add kernel partition instead of this change.
CONFIG_FLASH_PART_SIZE="1048,"
CONFIG_FLASH_PART_TYPE="smartfs,"
CONFIG_FLASH_PART_NAME="userfs,"

Reply:
done